README.md

```diff
@@ -38,6 +38,8 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/paritytec
 
 ## 🚀 Releases
 
+![Current Stable Release](https://img.shields.io/badge/Current%20Stable%20Release-polkadot_2407_1-green)&nbsp;&nbsp;![Next Stable Release (polkadot-2409)](https://img.shields.io/badge/Next%20Stable%20Release%20%28polkadot_2409%29-2024/09/25-orange)
```

Comment:
And you want to update them manually?

Reply:
At the beginning yes, as part of the version bumps backport routine, when release is done. Later we could, probably, automate it using the [release-registry](https://github.com/paritytech/release-registry) as a data source

Reply:
Okay, I hope you then ensure that this happens :D

Reply:
Actually, we can do this already now. Please use these URLs instead:  

`https://raw.githubusercontent.com/paritytech/release-registry/main/badges/polkadot-sdk-latest.svg`
`https://raw.githubusercontent.com/paritytech/release-registry/main/badges/polkadot-sdk-next.svg`

(test if they render here)
  
![a](https://raw.githubusercontent.com/paritytech/release-registry/main/badges/polkadot-sdk-latest.svg)
![b](https://raw.githubusercontent.com/paritytech/release-registry/main/badges/polkadot-sdk-next.svg)

Reply:
I've changed the URL's, looks good!